pkg/scaffold/api.go

```diff
@@ -212,14 +218,12 @@ func (api *API) scaffoldV2() error {
 			return fmt.Errorf("error updating kustomization.yaml: %v", err)
 		}
 
-		if !api.resourceExists() {
-			// update scaffolded resource in project file
-			api.project.Resources = append(api.project.Resources,
-				input.Resource{Group: r.Group, Version: r.Version, Kind: r.Kind})
-			err = saveProjectFile("PROJECT", api.project)
-			if err != nil {
-				fmt.Printf("error updating project file with resource information : %v \n", err)
-			}
```

Comment:
The check is needed here. It avoids having duplicates in the PROJECT file.

Reply:
Solved. Good point.